### PR TITLE
Refactored T1, T2

### PR DESCRIPTION
**Pull request by EdvinasKveselys**

### CODE REVIEW

File: cypress/integration/login.js [hunk 1]
```diff
@@ -1,94 +1,115 @@
...
+        loginPage.getLoginRole().click({force:true});
+        loginPage.getSpecificRoleFromDropDown("Admin").click();
+        loginPage.getSubmitButton().click();
+        loginPage.getUserInfoTitle("Edvinas Kveselys");
+        loginPage.getNavigationLength().should('have.length', 6);
+        loginPage.getActiveNavLinkColor("rgb(64, 76, 237)");
+        loginPage.getActiveNavLink("/time-logging");
     })
+
+    /*it('Admin creates new task', function() {
```
Comment (DavidV44) [inline cypress/integration/login.js]: Never commint commented out code, if it is commented chances are it should be deleted :)

File: cypress/integration/login.js [hunk 2]
```diff
@@ -1,94 +1,115 @@
...
+        loginPage.getSpecificRoleFromDropDown("Admin").click();
+        loginPage.getSubmitButton().click();
 
-        cy.url().should('include', '/time-logging');
-        cy.get('.page__title').contains('Timesheets')
-        cy.get('.calendar').should('be.visible')
-        cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Edvinas Kveselys');
-        cy.get('.main-nav').find('li').should('have.length', 6);
+        loginPage.urlShouldInclude("/time-logging");
```
Comment (DavidV44) [inline cypress/integration/login.js]: Methods with time logging elements should be put in `timeLoggingPage.js`. Also elements that are shared between pages (like navigation bar) should be put in separate file like `common.js` or `pageComponents.js`

**Review by DavidV44 — CHANGES_REQUESTED** [review]

File: cypress/integration/login.js [hunk 3]
```diff
@@ -1,94 +1,107 @@
...
+        common.getPageTitle("Timesheets");
+        timeLoggingPage.getCalendar().should('be.visible');
+        timeLoggingPage.getTileForm().should('be.visible');
+        common.getUserInfoTitle("Edvinas Kveselys");
+        common.getNavigationLength().should('have.length', 6);
 
-        const today = new Date();
-        const day = today.getDate();
-        cy.get('.calendar__date').contains(day);
+        timeLoggingPage.getTodaysDay();
```
Comment (nikolaj-tolkaciov) [inline cypress/integration/login.js]: you are getting today's day but not doing anything with it

File: cypress/integration/login.js [hunk 4]
```diff
@@ -1,94 +1,107 @@
...
+        common.visit();
+        loginPage.getLoginUserId().click({force:true});
+        loginPage.getSpecificUserFromDropDown("Edvinas Kveselys").click();
+        loginPage.getLoginRole().click({force:true});
+        loginPage.getSpecificRoleFromDropDown("User").click();
+        common.getSubmitButton().click();
+        common.getUserInfoTitle("Edvinas Kveselys");
+        common.getNavigationLength().should('have.length', 1);
+        common.getActiveNavLinkColor("rgb(64, 76, 237)");
+        common.getActiveNavLink("/time-logging");
```
Comment (nikolaj-tolkaciov) [inline cypress/integration/login.js]: you are getting active nav link but not doing anything with it

File: cypress/integration/login.js [hunk 5]
```diff
@@ -1,94 +1,107 @@
...
-        cy.get('.main-nav__link--active').should('have.attr', 'href', '/time-logging');
+        common.visit();
+        loginPage.getLoginUserId().click({force:true});
+        loginPage.getSpecificUserFromDropDown("Edvinas Kveselys").click();
+        loginPage.getLoginRole().click({force:true});
+        loginPage.getSpecificRoleFromDropDown("User").click();
+        common.getSubmitButton().click();
+        common.getUserInfoTitle("Edvinas Kveselys");
+        common.getNavigationLength().should('have.length', 1);
+        common.getActiveNavLinkColor("rgb(64, 76, 237)");
```
Comment (nikolaj-tolkaciov) [inline cypress/integration/login.js]: you are getting active nav link color but not doing anything with it

File: cypress/objects/loginPage.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
+class LoginPage{
+    
+
+    getUserValidationIndicator() {
```
Comment (nikolaj-tolkaciov) [inline cypress/objects/loginPage.js:3]: unnecessary gap above function name, one line would be enough

File: cypress/objects/timeLoggingPage.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+        return cy.get('.calendar')
+    }
+
+    getTileForm(){
+        return cy.get('.tile.form')
+    }
+
+    getTodaysDay(){
+        return cy.get('.calendar__date').contains(new Date().getDate())
+    }
```
Comment (nikolaj-tolkaciov) [inline cypress/objects/timeLoggingPage.js:12]: this function actually validates as well, not only getting it

Comment (nikolaj-tolkaciov) [reply cypress/objects/timeLoggingPage.js:12]: from your use case it does not seems like you are using what this function returns so maybe there is no need to return anything (remove unnecessary "return" )

**Review by nikolaj-tolkaciov — CHANGES_REQUESTED** [review]
Almost there! just few minor remarks

**Review by DavidV44 — APPROVED** [review]

**Pull request merged**